MonstieWash/Assets/Scripts/RoomSaver.cs

```diff
@@ -5,75 +5,69 @@
 
 public class RoomSaver : MonoBehaviour
 {
-    List<string> roomsLoaded;
-    Scene currentscene;
-    public GameObject testSpawner;
     public string firstRoomToLoad;
 
-    public void loadNewRoom(string target)
+    private List<string> roomsLoaded = new();
+    private Scene currentscene;
```

Comment:
I think these two private variables are missing a prefix indicating they are a member as per the Git Processes on the Dev Wiki